### ISSUE TITLE
Issues running Linux version

### ISSUE THREAD

**Issue by DuncanFaulkner** [nrwl/angular-console#200]
I know the linux version of this is in beta but doesn't seem to work as I would expect.

If I run ng serve --aot from a terminal the app builds and runs, using the same command in the UI console generates a sass-node issue, it suggests running npm rebuild node-sass and even though this completes it still doesn't build. 
If I then cancel this I don't appear to be able to build the app again as the port is in use and I have to quit the application and kill the remaining two processes before I can use the same port again. Am I missing something here?

I will do further testing on this as this may be an issue between keyboard and chair. Is there any documentation on how the application works? I know it should be self-explanatory as it is a UI over the command line but it just doesn't seem to work as I thought it might.

**Comment by FrozenPandaz** [comment]
The process is not exiting properly.

Possibly related to https://nodejs.org/api/child_process.html#child_process_subprocess_kill_signal

> Also note: on Linux, child processes of child processes will not be terminated when attempting to kill their parent. This is likely to happen when running a new process in a shell or with use of the shell option of ChildProcess:

**Comment by tptshoe** [comment]
I was having the same error as well. With regard to the error itself, it seems to be related to a mismatch in the node version on the machine:

> Error: Missing binding /home/mark/drfleggspm/spj-source/node_modules/node-sass/vendor/linux-x64-64/binding.node
> Node Sass could not find a binding for your current environment: Linux 64-bit with Node.js 10.x
> Found bindings for the following environments:
> - Linux 64-bit with Node.js 8.x

I have been using NVM and so I have multiple node versions on my machine. I switched to the system version (10.x) of node, then rebuilt node-sass (in my case with yarn add --force node-sass).

After that, serving my application from Angular Console works. It still leaves the process running after I exit Angular Console, but that looks like a separate issue.

Just for fun, I switched out to node v8.11.2, rebuilt node-sass, and tried again. Angular Console worked just fine, though it still left processes running and locked port 4200 after exit.
